core/src/utils/transition/index.ts

```diff
@@ -7,11 +7,14 @@ import {
   LIFECYCLE_WILL_LEAVE,
 } from '../../components/nav/constants';
 import type { NavOptions, NavDirection } from '../../components/nav/nav-interface';
+import { config } from '../../global/config';
```

Comment:
Should this use the `@global/config` import alias as well?

Reply:
Fixed in https://github.com/ionic-team/ionic-framework/pull/29400/commits/2c84f632c6d5848aa6949e48660b31e346e81fa7